concurrency-benchmarks.cabal

```diff
@@ -54,25 +71,27 @@ benchmark benchmarks
     transformers        >= 0.4   && < 0.6,
     async               >= 2.1.1 && < 2.3,
     random              >= 1.0   && < 2,
-    streamly            >= 0.4.0 && < 0.6
+    streamly
```

Comment:
Are you sure all streamly versions work with this package?

Reply:
No, I'm fixing the version of streamly via the env. `cabal.project`, `stack.yaml`. We can keep it rather simple. Developers will be the only ones using this. We don't need to support it over multiple versions.